run_robbie_viewer.sh

```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/env bash
+
+# run robbie viewer
+docker run -v ${PWD}/results:/data -p 5006:5006 robbie/robbie-viewer:latest
```

Comment:
I think it would be better if the results directory is an optional argument with a default. in case the user is running this from a different directory or put them in a strange spot.

Reply:
03ae791 adds an optional and default argument to the ``run_robbie_viewer.sh`` shell script.